docs/UserGuide.md

```diff
@@ -72,12 +72,12 @@ Wishful Shrinking is your desktop diet manager. It is an app that helps you **ma
 2. **Recommend recipes** to improve ease of home cooking 
 3. **Track daily food and calorie** intake<br><br>
 
-Wishful Shrinking targets **office workers** who tend to discount healthy eating. Office workers are also more
+Wishful Shrinking targets **office workers** who target on healthy eating. Office workers are also more
```

Comment:
er this phrasing makes it sound like we did not build the app **for** office workers instead we simply choose a group of people (eg office workers who target healthy eating) who will need our app. I think the previous phrasing is better to justify our target audience instead of modifying the target audience to suit the benefits of our app